connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/a2a/discovery/configuration/A2aDiscoveryConfigurationProperties.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.agenticai.a2a.discovery.configuration;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.bind.DefaultValue;
+import org.springframework.core.io.Resource;
+import org.springframework.validation.annotation.Validated;
+
+@Validated
+@ConfigurationProperties(prefix = "camunda.connector.agenticai.a2a.discovery")
```

Comment:
For now I use this prefix (ending with `discovery`) but it is not correct. I think the `discovery` package should be renamed to something like `tool` since we are doing more than just discovery. This holds for the MPC as well, I guess.